misc/python/materialize/mzbuild.py

```diff
@@ -258,6 +258,7 @@ def run(self) -> None:
                 s3.upload_fileobj(exe, self.bucket, exe_object_name)
             except NoCredentialsError:
                 print("Failed to find S3 credentials; not uploading build.")
+                return
             print(
                 f"Attempting to upload debug info to s3://{self.bucket}/{dbg_object_name}"
             )
```

Comment:
do we also need a try around the `s3.upload_fileobj` call right after this for `dbg`?

Reply:
I think the `return` ensures we don't get there? But I agree it's a bit confusing. Going to push up a commit to try to make it more clear.